### DISCUSSION TITLE
Synchronization between machines

### DISCUSSION THREAD

**Discussion by vns840** [alainm23/planify#1335] [Q&A]
Gostaria de parabenizar a ferramenta, pois ela é excelente. No entanto, tenho uma dúvida. Eu uso duas máquinas, uma no trabalho e outra em casa, e gostaria de manter as tarefas do Planify sincronizadas entre elas. Como posso fazer isso?

**Comment by dimas-cyriaco** [comment]
You can use something like syncthing to sync the database between machines. The file is on the folder `/home/username/.local/share/io.github.alainm23.planify/`.
